Add Tab for direct departures search

HerrLevin opened the pull request:
I've added a simple layout to search for departures at a single station instead of clicking through the planner.

https://github.com/user-attachments/assets/02ec8fc4-f187-424f-8491-9422da3da3d8

felixguendling commented:
Thank you! This looks like a useful feature!
The CI isn't green. Can you please check with `npm run check` and `npm run lint` locally?
Regarding readability, I would prefer to have more contrast than dark blue on black.
Is there a particular reason you didn't use the [shadcn tabs component](https://next.shadcn-svelte.com/docs/components/tabs)? Until now it has saved us some lines of code and styling work to just use shadcn.

HerrLevin commented:
Fixed the ci issues. :)

I'll look into the shadcn tabs!

traines-source commented on ui/src/lib/StopTimes.svelte:
```diff
@@ -53,7 +52,7 @@
 			variant="outline"
 			onclick={() => {
-				setArriveBy(!arriveBy);
+				onClickStop(stopId, stopName, queryTime, !arriveBy);
```
I think stopId and stopName arguments are in the wrong order, this also explains that I'm getting an error when switching to arrivals.

traines-source reviewed (changes requested):
Looking good in general, but
* I can't switch to arrivals anymore (error 500, see comment)
* mobile view is slightly destroyed due to fixed width
* on mobile, this further decreases the remaining space for the itinerary results, but I think this is something out of scope of this PR that needs to be improved/changed regardless
* I'm not 100% convinced of the tab name "Fahrplan/Timetable/Horaire". Maybe rather "Verbindungen/Connections/Itinéraires"? But that's just a suggestion.

HerrLevin commented:
Thanks for pointing this out! I thought I fixed the departures/arrivals bug already but apparently not. :D

- [x] fixed will only be applied after md breakpoint (Tried it with fractions but tailwind 3 is a bit limited here & this will give weird results. The fixed width seems to be the best solution above md. Below that `w-full` seems to do the trick)
- [x] fixed the departures/arrival thing
- [x] Changed the text to Connections (I like it!)

traines-source reviewed (approved):
Not sure why responsiveness wouldn't work similar to before, but it's good like that as well. Thanks :raised_hands:

networkException commented on ui/src/routes/+page.svelte:
```diff
@@ -338,28 +291,41 @@
 
 	<div class="maplibregl-control-container">
-		<div class="maplibregl-ctrl-top-left">
+		<div class="maplibregl-ctrl-top-left w-full">
```
Unfortunately this change causes the map to become non-interactive for the whole width of the page for any height range elements are visible on the left:

![image](https://github.com/user-attachments/assets/c8e67120-67bc-4a8f-9357-254d24362059)

![image](https://github.com/user-attachments/assets/42b553bf-0b22-4948-ba5a-02d850fcdafb)

Hovering anywhere over the highlighted area changes the cursor to `default` instead of `grab` and the map cannot be interacted with.

traines-source replied:
Thanks for noticing, I've fixed it in #837
